### PR TITLE
Improve cone scoring

### PR DESCRIPTION
**Pull request by BenG49**

### CODE REVIEW

File: src/main/java/com/stuypulse/robot/commands/arm/routines/ArmReady.java [hunk 1]
```diff
@@ -30,7 +30,14 @@ protected ArmTrajectory getTrajectory(ArmState src, ArmState dest) {
                 .addState(dest);
         }
 
-        return super.getTrajectory(src, dest);
+        double wristSafeAngle = 90; // src.getShoulderState().getCos() > 0 ? 120 : 60;
+
+        return new ArmTrajectory()
+            .addState(src.getShoulderDegrees(), wristSafeAngle)
+            .addState(dest.getShoulderDegrees(), wristSafeAngle)
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/commands/arm/routines/ArmReady.java:37]: this affects auton. Have an if !auton or something analogous around this, at least for midwest

File: src/main/java/com/stuypulse/robot/constants/Settings.java [hunk 2]
```diff
@@ -321,7 +321,7 @@ public interface Turn {
 
             SmartNumber MAX_TELEOP_TURNING = new SmartNumber("Driver Settings/Turn/Max Turning", 7.0);
 
-            SmartNumber MAX_SLOW_TURNING = new SmartNumber("Driver Settings/Turn/Max Slow Turning", Units.degreesToRadians(10));
+            SmartNumber MAX_SLOW_TURNING = new SmartNumber("Driver Settings/Turn/Max Slow Turning", 2.0);
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/constants/Settings.java:324]: 100 degrees per second?

File: src/main/java/com/stuypulse/robot/RobotContainer.java [hunk 3]
```diff
@@ -136,7 +136,8 @@ private void configureDriverBindings() {
 
         // swerve
         driver.getLeftButton()
-            .whileTrue(new ManagerChooseScoreSide().andThen(new SwerveDriveToScorePose()));
+            .whileTrue(new SwerveDriveSlowDrive(driver));
+            // .whileTrue(new ManagerChooseScoreSide().andThen(new SwerveDriveToScorePose()));
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/RobotContainer.java:140]: what's the align button for the future then? What if this was slow drive in cone mode and align in cube mode?

File: src/main/java/com/stuypulse/robot/constants/Settings.java [hunk 4]
```diff
@@ -305,12 +305,12 @@ public interface Drive {
...
 
             SmartNumber RC = new SmartNumber("Driver Settings/Drive/RC", 0.25);
-            SmartNumber POWER = new SmartNumber("Driver Settings/Drive/Power", 2);
+            SmartNumber POWER = new SmartNumber("Driver Settings/Drive/Power", 3);
 
             SmartNumber MAX_TELEOP_SPEED = new SmartNumber("Driver Settings/Drive/Max Speed", Units.feetToMeters(15.67));
             SmartNumber MAX_TELEOP_ACCEL = new SmartNumber("Driver Settings/Drive/Max Accleration", 6.5);
 
-            SmartNumber MAX_SLOW_SPEED = new SmartNumber("Driver Settings/Drive/Max Slow Speed", Units.feetToMeters(1));
+            SmartNumber MAX_SLOW_SPEED = new SmartNumber("Driver Settings/Drive/Max Slow Speed", Units.feetToMeters(2.0));
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/constants/Settings.java:313]: I don't know how fast you want, but this is _a _bit__ faster than that default charging station auto balance speed (0.6m/s vs 0.5m/s)

**Review by selym3 — APPROVED** [review]

File: src/main/java/com/stuypulse/robot/RobotContainer.java (hunk 3 above)
Comment (ShauryaSen) [reply src/main/java/com/stuypulse/robot/RobotContainer.java:140]: sure that works

**Review by selym3 — APPROVED** [review]

**Pull request merged**